data/data-pipeline/data_pipeline/tests/sources/national_risk_index/test_etl.py

```diff
@@ -137,11 +119,11 @@ def test_get_output_file_path(self, mock_etl, mock_paths):
         )
         assert output_file_path == expected_output_file_path
 
-    def test_extract(self, mock_etl, mock_paths):
-        """Ensure the extract results are working as appropriate."""
+    def test_extract_path(self, mock_etl, mock_paths):
+        """Ensure the extract is happening where appropriate."""
+        # this is returning true for no reason?
```

Comment:
What does this comment `# this is returning true for no reason?` mean?

Reply:
yikes, this is a comment i left in from when i was implementing, removed.